Warnings in Test_FirstOrderComputeTags.cpp

# Bug reports:

```cpp
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:98:15: warning: unused variable 'fluxes' [-Wunused-variable]
  const auto& fluxes =
              ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:128:3: note: in instantiation of function template specialization '(anonymous namespace)::test_first_order_compute_tags<1>' requested here
  test_first_order_compute_tags<1>();
  ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:117:15: warning: unused variable 'sources' [-Wunused-variable]
  const auto& sources = get<db::add_tag_prefix<::Tags::Source, vars_tag>>(box);
              ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:98:15: warning: unused variable 'fluxes' [-Wunused-variable]
  const auto& fluxes =
              ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:129:3: note: in instantiation of function template specialization '(anonymous namespace)::test_first_order_compute_tags<2>' requested here
  test_first_order_compute_tags<2>();
  ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:117:15: warning: unused variable 'sources' [-Wunused-variable]
  const auto& sources = get<db::add_tag_prefix<::Tags::Source, vars_tag>>(box);
              ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:98:15: warning: unused variable 'fluxes' [-Wunused-variable]
  const auto& fluxes =
              ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:130:3: note: in instantiation of function template specialization '(anonymous namespace)::test_first_order_compute_tags<3>' requested here
  test_first_order_compute_tags<3>();
  ^
../tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp:117:15: warning: unused variable 'sources' [-Wunused-variable]
  const auto& sources = get<db::add_tag_prefix<::Tags::Source, vars_tag>>(box);
              ^
6 warnings generated.
```

--- a/tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp
+++ b/tests/Unit/Elliptic/Test_FirstOrderComputeTags.cpp
@@ -95,9 +95,6 @@ void test_first_order_compute_tags() {
           std::move(vars), FluxesComputer{}, 3.);
 
   // Check computed fluxes
-  const auto& fluxes =
-      get<db::add_tag_prefix<::Tags::Flux, vars_tag, tmpl::size_t<Dim>,
-                             Frame::Inertial>>(box);
   for (size_t d = 0; d < Dim; d++) {
     CHECK(get<::Tags::Flux<FieldTag, tmpl::size_t<Dim>, Frame::Inertial>>(box)
               .get(d) == DataVector{num_points, 3. * (d + 1.)});
@@ -114,7 +111,6 @@ void test_first_order_compute_tags() {
   }
 
   // Check computed sources
-  const auto& sources = get<db::add_tag_prefix<::Tags::Source, vars_tag>>(box);
   CHECK(get(get<::Tags::Source<FieldTag>>(box)) == DataVector{num_points, 18.});
   for (size_t d = 0; d < Dim; d++) {
     CHECK(get<::Tags::Source<AuxiliaryFieldTag<Dim>>>(box).get(d) ==

